flink-connectors/flink-connector-kinesis/src/main/java/org/apache/flink/streaming/connectors/kinesis/metrics/ShardMetricsReporter.java

```diff
@@ -28,14 +28,14 @@
 public class ShardMetricsReporter {
 
 	private volatile long millisBehindLatest = -1;
-	private volatile double loopFrequencyHz = 0.0;
-	private volatile double bytesPerRead = 0.0;
-	private volatile long runLoopTimeNanos = 0L;
-	private volatile long averageRecordSizeBytes = 0L;
-	private volatile long sleepTimeMillis = 0L;
-	private volatile int numberOfAggregatedRecords = 0;
-	private volatile int numberOfDeaggregatedRecords = 0;
-	private volatile int maxNumberOfRecordsPerFetch = 0;
+	private volatile double loopFrequencyHz = -1.0;
+	private volatile double bytesPerRead = -1.0;
+	private volatile long runLoopTimeNanos = -1;
+	private volatile long averageRecordSizeBytes = -1;
+	private volatile long sleepTimeMillis = -1;
```

Comment:
negative default values are discouraged as they can mess with aggregations.

Reply:
Agreed that they can mess with aggregations. I changed the default values to 0 and made corresponding doc updates.
Note: `millisBehindLatest` (not a part of this PR) has a default value of -1.